etna/media/forms.py

```diff
@@ -0,0 +1,64 @@
+import magic
+
+from django import forms
+from django.core.exceptions import ValidationError
+
+from wagtail.admin import widgets
+
+from wagtailmedia.permissions import permission_policy as media_permission_policy
+from wagtail.admin.forms.collections import BaseCollectionMemberForm, collection_member_permission_formset_factory
+
+from django.forms.models import modelform_factory
+
+
+class BaseMediaForm(BaseCollectionMemberForm):
+    class Meta:
+        widgets = {
+            "date": widgets.AdminDateInput,
+            "tags": widgets.AdminTagWidget,
+            "file": forms.FileInput,
+        }
+
+    permission_policy = media_permission_policy
+
+    def __init__(self, *args, **kwargs):
+        super(BaseMediaForm, self).__init__(*args, **kwargs)
+
+        # Remove unused fields.
+        for name in ("width", "height", "thumbnail"):
+            # these fields might be editable=False so verify before accessing
```

Comment:
What do you mean by this? It doesn't look like we're checking for `editable=False`

Reply:
Done